content/server/soju.md

```diff
@@ -0,0 +1,194 @@
+
+Make an admin user called anon:
+
+```sh
+sojudb create-user anon -admin
+```
+
+### Systemd Service
+
+Then let's make a service in /etc/systemd/system/soju.service
+
+```systemd
+[Unit]
+Description=soju IRC bouncer service
+Documentation=https://soju.im/
+Documentation=man:soju(1) man:sojuctl(1)
+Wants=network-online.target
+After=network-online.target
+
+[Service]
+Type=simple
+User=soju
+Group=soju
+DynamicUser=yes
+StateDirectory=soju
+ConfigurationDirectory=soju
+RuntimeDirectory=soju
+AmbientCapabilities=CAP_NET_BIND_SERVICE
+ExecStart=/usr/local/bin/soju
+ExecReload=/bin/kill -HUP $MAINPID
+Restart=on-failure
+
+[Install]
+WantedBy=multi-user.target
+```
+
+Then enable our new service
+
+```sh
+systemctl daemon-reload
+systemctl enable --now soju
+```
+
+## Usage
+
+### Client Configurations
+
+Soju can be used by most legacy irc clients, but ones that support the soju.im/bouncer-networks extension make multiple servers work seamlessly.
+
+Here is a table of recommended clients that support said extension.
+
+| Client                                                      | Interface    |
+| :---------------------------------------------------------- | :----------  |
+| [Senpai](https://sr.ht/~delthas/senpai/)                    | Terminal/TUI |
+| [Goguma](https://f-droid.org/packages/fr.emersion.goguma/)  | Android/iOS  |
+| [Gamja](https://sr.ht/~emersion/gamja/)                     | Web          |
+| [Chathistorysync](https://sr.ht/~emersion/chathistorysync/) | Plain Text   |
+
+See more client configurations [here](https://git.sr.ht/~emersion/soju/tree/master/item/contrib/clients.md) for legacy clients.
+
+Here is an example client configuration for senpai.
+
+```scfg
+address  {{<hl>}}irc.example.org{{</hl>}}
+nickname anon
+password <password for anon>
+```
+
+### New Networks
+
+Several of the clients above have special user interfaces to add new servers, but the default way to do it with any client is with the `network create` command.
+
+For example, this is how we'd add [libera.chat](https://libera.chat) to our server:
+
+```
+/msg BouncerServ net create -addr irc.libera.chat -name libera -realname "mr anon"
+```
+
+### New Users
+
+Your bouncer can be used by friends too. To set up a user, an admin user can run the following:
+
+```
+/msg BouncerServ user create <username> -password <password>
+```
+
+Note: If you set up the `unix+admin://` listen directive in your config, you can run any of the previous commands with `sojuctl <BouncerServ cmd>` in a shell.
+
+## Further Configuration
+
+You should have a usable bouncer by now, but there are more config options worth looking into. Read the documentation from soju's man page or the [contrib](https://git.sr.ht/~emersion/soju/tree/master/item/contrib/) section of the repo.
+
+Of note:
+
+* websocket listeners to configure a local gamja server
+
+* file-upload protocol and drivers
+
+* motd if you wanna show off some cool [ascii art](https://www.asciiart.eu/computers/linux)
```

Comment:
What purpose does this section serve? It's a bit unclear to anyone reading it

Reply:
It's a good habit to keep the ending of these guides open-ended so people read more docs or features they might wanna have. The first bullet point is also residue from my link to a nonexistent gamja guide I have written out.

Reply:
the file-upload protocol stuff is probably a bit much though, so if you have a better suggestion for the ending, please share your thoughts.